### ISSUE TITLE
Platform does not start anymore

### ISSUE THREAD

**Issue by bartek75** [Chouffy/home_assistant_tgtg#147]
## System Information

version | core-2024.11.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.4
os_name | Linux
os_version | 6.6.54-haos
arch | x86_64
timezone | Europe/Warsaw
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4906
Installed Version | 1.34.0
Stage | running
Available Repositories | 1456
Downloaded Repositories | 5

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 13.2
-- | --
update_channel | stable
supervisor_version | supervisor-2024.11.4
agent_version | 1.6.0
docker_version | 27.2.0
disk_total | 30.8 GB
disk_used | 9.6 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | kvm
board | ova
supervisor_api | ok
version_api | ok
installed_addons | Advanced SSH & Web Terminal (19.0.0), File editor (5.8.0), MariaDB (2.7.1), Samba share (12.3.2), Mosquitto broker (6.4.1), Zigbee2MQTT (1.41.0-1), Home Assistant Google Drive Backup (0.112.1), Cloudflared (5.2.2)

</details>

<details><summary>Dashboards</summary>

dashboards | 2
-- | --
resources | 1
views | 4
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | November 19, 2024 at 10:41
-- | --
current_recorder_run | November 25, 2024 at 19:00
estimated_db_size | 1223.23 MiB
database_engine | mysql
database_version | 10.11.6

</details>

<details><summary>Sonoff</summary>

version | 3.6.0 (8dd8af9)
-- | --
cloud_online | 6 / 8
local_online | 6 / 6

</details>

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/Chouffy/home_assistant_tgtg/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Tgtg does not start anymore.
All tgtg entities are unavailable
It might be the  same issue as in here: https://github.com/Der-Henning/tgtg/issues/556

### Reproduction steps

1. Restart Homeassistant

### Debug logs

```text
2024-11-25 19:00:54.244 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up tgtg platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 365, in _async_setup_platform
    await asyncio.shield(awaitable)
  File "/usr/local/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/tgtg/sensor.py", line 83, in setup_platform
    tgtgReply = tgtg_client.get_items()
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/tgtg/__init__.py", line 244, in get_items
    self.login()
  File "/usr/local/lib/python3.12/site-packages/tgtg/__init__.py", line 153, in login
    self._refresh_token()
  File "/usr/local/lib/python3.12/site-packages/tgtg/__init__.py", line 139, in _refresh_token
    raise TgtgAPIError(response.status_code, response.content)
tgtg.exceptions.TgtgAPIError: (404, b'{"type":"about:blank","title":"Not Found","status":404,"detail":"No static resource auth/v3/token/refresh.","instance":"/api/auth/v3/token/refresh"}')
```

**Comment by nunodxxd** [comment]
same here

**Comment by tjorim** [comment]
Fixed in version 5.6.0: https://github.com/Chouffy/home_assistant_tgtg/releases/tag/5.6.0
